src/main/java/de/retest/web/selenium/RecheckDriver.java

```diff
@@ -0,0 +1,273 @@
+		return ((FindsByLinkText) wrapped).findElementsByLinkText( using );
+	}
+
+	@Override
+	public WebElement findElementByPartialLinkText( final String using ) {
+		return ((FindsByLinkText) wrapped).findElementByPartialLinkText( using );
+	}
+
+	@Override
+	public List<WebElement> findElementsByPartialLinkText( final String using ) {
+		return ((FindsByLinkText) wrapped).findElementsByPartialLinkText( using );
+	}
+
+	@Override
+	public WebElement findElementByName( final String using ) {
+		return ((FindsByName) wrapped).findElementByName( using );
+	}
+
+	@Override
+	public List<WebElement> findElementsByName( final String using ) {
+		return ((FindsByName) wrapped).findElementsByName( using );
+	}
+
+	@Override
+	public WebElement findElementByCssSelector( final String using ) {
+		return ((FindsByCssSelector) wrapped).findElementByCssSelector( using );
+	}
+
+	@Override
+	public List<WebElement> findElementsByCssSelector( final String using ) {
+		return ((FindsByCssSelector) wrapped).findElementsByCssSelector( using );
+	}
+
+	@Override
+	public WebElement findElementByTagName( final String using ) {
+		return ((FindsByTagName) wrapped).findElementByTagName( using );
+	}
+
+	@Override
+	public List<WebElement> findElementsByTagName( final String using ) {
+		return ((FindsByTagName) wrapped).findElementsByTagName( using );
+	}
+
+	@Override
+	public WebElement findElementByXPath( final String using ) {
+		return ((FindsByXPath) wrapped).findElementByXPath( using );
+	}
+
+	@Override
+	public List<WebElement> findElementsByXPath( final String using ) {
+		return ((FindsByXPath) wrapped).findElementsByXPath( using );
+	}
+
+	@Override
+	public void close() {
+		wrapped.close();
+	}
+
+	@Override
+	public void get( final String url ) {
+		wrapped.get( url );
+	}
+
+	@Override
+	public String getCurrentUrl() {
+		return wrapped.getCurrentUrl();
+	}
+
+	@Override
+	public String getPageSource() {
+		return wrapped.getPageSource();
+	}
+
+	@Override
+	public String getTitle() {
+		return wrapped.getTitle();
+	}
+
+	@Override
+	public String getWindowHandle() {
+		return wrapped.getWindowHandle();
+	}
+
+	@Override
+	public Set<String> getWindowHandles() {
+		return wrapped.getWindowHandles();
+	}
+
+	@Override
+	public Options manage() {
+		return wrapped.manage();
+	}
+
+	@Override
+	public Navigation navigate() {
+		return wrapped.navigate();
+	}
+
+	@Override
+	public void quit() {
+		wrapped.quit();
+	}
+
+	@Override
+	public TargetLocator switchTo() {
+		return wrapped.switchTo();
+	}
+
+	@Override
+	public Object executeScript( final String script, final Object... args ) {
+		return ((JavascriptExecutor) wrapped).executeScript( script, args );
+	}
+
+	@Override
+	public Object executeAsyncScript( final String script, final Object... args ) {
+		return ((JavascriptExecutor) wrapped).executeAsyncScript( script, args );
+	}
+
+	@Override
+	public <X> X getScreenshotAs( final OutputType<X> target ) throws WebDriverException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the declaration of thrown exception 'org.openqa.selenium.WebDriverException' which is a runtime exception. ([squid:RedundantThrowsDeclarationCheck](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3ARedundantThrowsDeclarationCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=77&issues=AWaHMZUnlB6-zLfzofGB&open=AWaHMZUnlB6-zLfzofGB)

[ISSUE_KEY]: AWaHMZUnlB6-zLfzofGB